babel.config.js

```diff
@@ -1,5 +1,23 @@
 const plugins = [
-  '@babel/plugin-transform-modules-commonjs',
+  [
+    '@babel/plugin-transform-runtime',
+    {
+      helpers: true,
+      // regenerator: true,
```

Comment:
I think that `true` is the default value.

Reply:
noted. i don't want to change it now, because I'd have to re-test everything, but i'll pay closer attention next time i'm looking at this